### PR TITLE
Edittor retouches

### PR DESCRIPTION
**Pull request by manjav**

### CODE REVIEW

File: editor/lib/display/context_menu.dart
Comment (tavangar79) [inline editor/lib/display/context_menu.dart:1]: we can use popupmenu 
https://api.flutter.dev/flutter/material/PopupMenuButton-class.html

Comment (manjav) [reply editor/lib/display/context_menu.dart:1]: Thanks for your offer

Comment (manjav) [reply editor/lib/display/context_menu.dart:1]: Thanks for your offer

Comment (manjav) [reply editor/lib/display/context_menu.dart:1]: Thanks for your offer

**Pull request merged**